packages/cli/src/cliEntry.js

```diff
@@ -142,7 +142,13 @@ async function setupAndRun() {
     ? path.join('..', 'setup_env.bat')
     : path.join('..', 'setup_env.sh');
 
-  childProcess.execFileSync(path.join(__dirname, setupEnvScript));
+  try {
+    childProcess.execFileSync(path.join(__dirname, setupEnvScript));
+  } catch (error) {
+    logger.warn(
+      `Failed to run environment setup script, but we can continue without it. You may want to apply the steps manually.\n${error}`
```

Comment:
This isn't super descriptive and doesn't tell people what to do to fix it either. What about something like:
```
Failed to run environment setup script (`setup_env.sh|bat`). This script can continue to run if your local environment matches what React Native expects. If it does fail, check out `path/to/setup_env` and adjust your environment to match it.
```

?

Reply:
Oh yea, way better wording. Btw while doing that I noticed that the windows setup_env script is (and has always been) a file with only comments. What's the idea behind it?

Reply:
It may have just been a placeholder. Feel free to kill it and ignore this codepath on Windows altogether.

Reply:
Adjusted the wording and removed Windows script ✨